cads_adaptors/tools/url_tools.py

```diff
@@ -8,6 +8,7 @@
 import jinja2
 import multiurl
 import requests
+import yaml
```

Comment:
pyyaml is not python standard library. Looks like another dependency is installing it, but that's not reliable.
If you need to use pyyaml, you should add it to the`environment.yml` and to the list of mandatory dependencies in `pyproject.toml`. It's only used to format an error message though. Is it worth it?

Reply:
nxt commit